app/controllers/todo_controller.rb

```diff
@@ -0,0 +1,9 @@
+class TodoController < ApplicationController
```

Comment:
Model, Controller and View are usually tied to one resource.
In this case you have `TodoController` but it should be `TasksController`, managing only one resource - Tasks